src/test/java/com/google/firebase/database/FirebaseDatabaseTest.java

```diff
@@ -210,9 +212,14 @@ public void testDbUrlIsEmulatorUrlWhenSettingOptionsManually() {
         new CustomTestCase("https://test.firebaseio.com?ns=valid-namespace", "localhost:90",
             "http://localhost:90", "valid-namespace")
     );
+
+    GoogleCredentials credentials = GoogleCredentials.fromStream(ServiceAccount.EDITOR.asStream());
+    FirebaseOptions options = FirebaseOptions.builder()
+        .setCredentials(credentials)
+        .build();
```

Comment:
Minor nit: We can put this in a helper method and re-use across tests.

Reply:
Good idea. Refactored the options into a static constant.